### DISCUSSION TITLE
How to always init on startup in netbox-docker?

### DISCUSSION THREAD

**Discussion by steffenschumacher** [tobiasge/netbox-initializers#16] [Q&A] [answered]
I do see this ```./manage.py load_initializer_data --path /path/for/example/files``` in the readme.
I don't know enough about plugins to know if I'm supposed to either append this command to the docker file or tweak the configuration.py to make sure this command is always run on startup of netbox?

Perhaps the readme could include more details on this, or am I just missing the obvious?

**Comment by tobiasge** [comment, marked as answer]
If you have your own fork of `netbox-docker` you could place a call to the import at the end  of `docker-entrypoint.sh` just before the `exec`.

**Reply by steffenschumacher** [reply]
Right - makes sense👍
